src/types/index.ts

```diff
@@ -12,5 +28,14 @@ export type SubComponentProps<P> = Partial<P> & {
   // https://github.com/microsoft/TypeScript/issues/28960
   [prop: string]: unknown;
 };
+/**
+ * This type is simply an alias for the 'any' type and should be used sparingly,
+ * if at all.
+ */
 export type TSFixMe = any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I actually have a feeling we don't use that anywhere anymore.

Reply:
Yeah we don't actually, but I feel like we should keep it just to be safe. Might be handy for onboarding people unfamiliar with TS for example.